admin-client/src/pages/organization/Index.svelte

```diff
@@ -1,7 +1,21 @@
 <script>
-  import { fetchOrganizations } from '../../lib/api';
+  import page from 'page';
+  import { fetchOrganizations, deactivateOrganization, activateOrganization } from '../../lib/api';
   import { formatDateTime } from '../../helpers/formatter';
   import { DataTable, PaginatedQueryPage } from '../../components';
+
+  function deactivate(id) {
+    // eslint-disable-next-line no-alert
+    if (window.confirm('Are you sure you want to deactivate this organization?')) {
+      deactivateOrganization(id);
+      page('/organizations');
+    }
+  }
```

Comment:
We probably want to wait for the deactivate before navigating. Also, what happens if it fails?

```suggestion
  async function deactivate(id) {
    // eslint-disable-next-line no-alert
    if (window.confirm('Are you sure you want to deactivate this organization?')) {
      await deactivateOrganization(id);
      page('/organizations');
    }
  }
```

Reply:
I went further, just pushed: 

```javascript
  async function deactivate(id) {
    // eslint-disable-next-line no-alert
    if (!window.confirm('Are you sure you want to deactivate this organization?')) { return null; }
    try {
      await deactivateOrganization(id);
      page('/organizations');
      return notification.setSuccess(`Organization ${id} deactivated successfully!`);
    } catch (error) {
      return notification.setError(`Unable to deactivate organization ${id}: ${error.message}`);
    }
  }
```